testing/ossp-uuid/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Contributor: "Andrzej Trzaska <atrzaska2@gmail.com>"
+# Maintainer: "Andrzej Trzaska <atrzaska2@gmail.com>"
+pkgname=ossp-uuid
+pkgver=1.6.2
+pkgrel=0
+pkgdesc="Open Source Software Project Universally Unique Identifier (UUID)"
+url="http://www.ossp.org/pkg/lib/uuid/"
+arch="all"
+license="MIT"
+subpackages="$pkgname-doc"
+source="https://mirrors.ocf.berkeley.edu/debian/pool/main/o/ossp-uuid/ossp-uuid_1.6.2.orig.tar.gz"
+builddir="$srcdir/uuid-$pkgver"
+
+build() {
+	cd "$builddir"
+	./configure --prefix=/usr --without-pgsql --without-perl --without-php
```

Comment:
sadly it is not compatible with php7, but fine

Reply:
What about other integrations (pgsql & perl)? Are they useful?

Reply:
They are not compiled by default in brew formula https://github.com/Homebrew/homebrew-core/blob/master/Formula/ossp-uuid.rb, so probably they are not that useful